pymatgen/analysis/tests/test_phase_diagram.py

```diff
@@ -222,7 +235,7 @@ def test_get_formation_energy(self):
             self.assertAlmostEqual(energy, stable_formation_energies[formula], 7)
 
     def test_all_entries_hulldata(self):
-        self.assertEqual(len(self.pd.all_entries_hulldata), 492)
+        self.assertEqual(len(self.pd.all_entries_hulldata), 490)
```

Comment:
I'm assuming this changed because two entries are now equivalent.

Reply:
Yes in the test file we have two sets of duplicated entries. "Li2O,2,0,1,-14.31361175" on both line 8 and line 14 and "Fe,0,1,0,-6.59394176" is repeated on lines 11 and 19.